lib/segment/benches/hnsw_build_asymptotic.rs

```diff
@@ -71,16 +72,18 @@ fn hnsw_build_asymptotic(c: &mut Criterion) {
         })
     });
 
+    let mut scored_points_buffer = vec![ScoredPointOffset { idx: 0, score: 0. }; 1000];
```

Comment:
I see `ScoredPointOffset { idx: 0, score: 0. }` quite often in this diff.
Maybe `ScoredPointOffset` could implement `Default`?